context.go

```diff
@@ -623,41 +623,42 @@ func (c *Context) ParseFileList(rootDir string, filePaths []string) (deps []stri
 	var numErrs uint32
 	var numGoroutines int32
 
-	// handler must be reentrant
+	// handlers must be reentrant
+	fileCounter := func(file *parser.File) {
+		atomic.AddInt32(&numGoroutines, 1)
+	}
 	handleOneFile := func(file *parser.File) {
+
```

Comment:
I generally prefer no blank line at the beginning of a function, unless the arguments to the function wraps to multiple lines.

Reply:
oops